tests/test_disjunction.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
```

Comment:
Thank you for including a new test with this!

Reply:
you are welcome :)